doc/release/1.13.0-notes.rst

```diff
@@ -7,6 +7,8 @@ This release supports Python 2.7 and 3.4 - 3.6.
 Highlights
 ==========
 
+ * Operations like `a + b + c` will create less temporaries on some platforms
```

Comment:
needs double backticks, here and in all cases (including `` `backtrace` ``) below